attributes/network/cddl/4_ledger_transactions.cddl

```diff
@@ -58,14 +58,33 @@ transaction-id = bstr
 
 ; Transaction types. This can be extended and the server MUST provide a list of
 ; those types.
-; TODO: namespace transaction types per attribute ID.
-transaction-type = transaction-send
+; Extensible transaction info.
+; TODO: namespace transaction types per attribute ID, somehow.
+transaction-info = transaction-send
                  / transaction-mint
                  / transaction-burn
 
-transaction-send = 0
-transaction-mint = 1
-transaction-burn = 2
+transaction-send = [
```

Comment:
Is there a way to re-use what's in `6_ledger_commands.cddl`?

Reply:
We will, as a separate PR. I have a few tooling ideas around the CDDL, including generating API documentation.